pipelines/protocolB.yaml

```diff
@@ -0,0 +1,278 @@
+      doDefect: parameters.defects
+      doSaturationInterpolation: false
+      growSaturationFootprintSize: 0
+  cptMeasureDefects:
+    class: lsst.cp.pipe.defects.MeasureDefectsTask
+    config:
+      connections.inputExp: 'cptDefectsProc'
+      connections.outputDefects: 'cptPartialDefects'
+  cptMergeDefects:
+    class: lsst.cp.pipe.defects.MergeDefectsTask
+    config:
+      connections.inputDefects: 'cptPartialDefects'
+      connections.mergedDefects: 'faintDefects'
+
+  # # PTC:
+  cptPtcIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'flat'
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cptPtcProc'
+      connections.bias: 'bias'
+      connections.dark: 'dark'
+      connections.flat: 'flat'
+      doWrite: true
+      doOverscan: true
+      doAssembleCcd: true
+      doBias: true
+      doVariance: true
+      doLinearize: parameters.linearize
+      doCrosstalk: parameters.crosstalk
+      doBrighterFatter: false
+      doDark: true
+      doStrayLight: false
+      doFlat: false
+      doFringe: false
+      doApplyGains: false
+      doDefect: parameters.defects
+      doNanMasking: true
+      doInterpolate: false
+      doSaturation: false
+      doSaturationInterpolation: false
+      growSaturationFootprintSize: 0
+  cptPtcExtract:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveExtractTask
+    config:
+      connections.inputExp: 'cptPtcProc'
+      connections.taskMetadata: 'cptPtcIsr_metadata'
+      connections.outputCovariances: 'cptPtcExtract'
+      matchExposuresType: "EXPID"
+  cptPtcSolve:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveSolveTask
+    config:
+      connections.inputCovariances: 'cptPtcExtract'
+      connections.outputPtcDataset: 'ptc'
+      ptcFitType: EXPAPPROXIMATION
+
+  #linearity
+  cptLinearitySolve:
+    class: lsst.cp.testing.CptLinearitySolveTask
+    config:
+      connections.inputPtc: 'ptc'
+      connections.outputLinearizer: 'linearizer'
+      linearityType: 'Spline'
+
+  #bfk
+  cptBfkSolve:
+    class: lsst.cp.testing.CptBrighterFatterKernelSolveTask
+    config:
+      connections.inputPtc: 'ptc'
+      connections.outputBFK: 'brighterFatterKernel'
+
+subsets:
+  # Single calibration subsets.  These may be better run with the
+  # standard cp_pipe pipelines.
+  bias:
+    subset:
+      - cptBiasIsr
+      - cptBiasCombine
+  dark:
+    subset:
+      - cptDarkIsr
+      - cptDark
+      - cptDarkCombine
+  flat:
+    subset:
+      - cptFlatIsr
+      - cptFlatMeasure
+      - cptFlatNorm
+      - cptFlatCombine
+  brightDefects:
+    subset:
+      - cptBrightDefectIsr
+      - cptMeasureBrightDefects
+      - cptMergeBrightDefects
+  defects:
+    subset:
+      - cptDefectIsr
+      - cptMeasureDefects
+      - cptMergeDefects
+  ptc:
+    subset:
+      - cptPtcIsr
+      - cptPtcExtract
+      - cptPtcSolve
+  postPtc:
+    subset:
+      - cptLinearitySolve
+      - cptBfkSolve
+# contracts:
```

Comment:
Why are these commented out?

Reply:
I do not recall.  They all seem valid, so I've made sure they apply.